source/tutorials/filtering.rst

```diff
@@ -129,6 +138,15 @@ For example, filtering the table to contain only samples from subject 1 is perfo
       --p-where "Subject='subject-1' AND BodySite='gut'" \
       --o-filtered-table subject-1-gut-filtered-table.qza
 
+The ``OR`` keyword syntax is similar to the ``AND`` keyword syntax, but specifies that either of the expressions can be true for a sample to be retained. For lack of a more relevant application to the example data being used here, the ``OR`` keyword in this example is applied to retain samples all of the samples where ``BodySite`` is ``gut`` *or* ``ReportedAntibioticUsage`` is ``Yes`` in ``sample-metadata.tsv``.
```

Comment:
Any suggestions on a more relevant use of the ``OR`` keyword here? I specifically want one where the metadata category is different in the two expressions (i.e., I don't want ``BodySite='left palm' OR BodySite='right palm'`` as that's better achieved with ``IN``).